.github/workflows/create-tag.yml

```diff
@@ -1,16 +1,18 @@
 name: db-syncer -Create Tag
```

Comment:
I think we should rename this action to `publish-release.yaml`  and update the `name` field accordingly as it's more descriptive of the workflows purpose.

Reply:
name: Gingersnap db-syncer - Create tag and publish